lib/node_modules/@stdlib/blas/base/sscal/src/sscal.c

```diff
@@ -0,0 +1,69 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* Multiply `X` by a constant.
+*/
+#include "sscal.h"
+
+/**
+* Multiplies `X` by a constant.
+*
+* @param N       number of indexed elements
+* @param alpha   scalar
+* @param X       input array
+* @param stride  index increment
+*/
+void c_sscal( const int N, const float alpha, float *X, const int stride ) {
+	int ix;
+	int i;
+	int m;
+
+	if ( N <= 0 || stride <= 0) {
+		return;
+	}
+
+	if ( alpha == 0.0 ) {
```

Comment:
This branch should be removed.

Reply:
Correction. This should be `alpha == 1.0`. This applies here and elsewhere where `alpha == 0.0` was included.